sklearn/tests/test_multioutput.py

```diff
@@ -174,6 +174,20 @@ def test_multi_output_classification_partial_fit_parallelism():
         assert_false(est1 is est2)
 
 
+# check predict_proba passes
+def test_multi_output_predict_proba():
+    sgd_linear_clf = SGDClassifier(loss='log', random_state=1, max_iter=5)
+    multi_target_linear = MultiOutputClassifier(sgd_linear_clf)
+    multi_target_linear.fit(X, y)
+    multi_target_linear.predict_proba(X)
+
+    sgd_linear_clf = SGDClassifier(random_state=1, max_iter=5)
```

Comment:
Could you please add a inline comment before this line to make it explicit that `SGDClassifier` uses `loss='hinge' by default which is not a probabilistic loss function and therefore does not expose a `predict_proba` method.`

Reply:
> I don't understand "create a new estimator for the test that only has predict_proba after fitting": this is already the case for SGDClassifier(loss='log'), no?

No, as @TomDLT said above, this is not actually the case for ``SGDClassifier`` in master, and the current test is not failing in master.